django_project/ta_system/admin.py

```diff
@@ -187,7 +210,7 @@ class ProfileAdmin(ModelAdmin):
             'fields': ('courses_taken',)
         }),
         ('Edit Student Information', {
-            'fields': ('user', 'eagle_id')
+            'fields': ('user', 'eagle_id', 'lab_hour_preferences')
```

Comment:
We should undo this change, since it was just for debugging purposes. We'll come up with a better way to display each student's lab hour preferences and lab hour assignments in future tickets.

Reply:
code deleted